backend/src/validators/projects/closeout.js

```diff
@@ -12,10 +12,22 @@ const getOneById = {
       .prop("records_filed", Schema.Picker)
       .prop("contract_ev_completed", Schema.Picker)
       .prop("contractor_security_terminated", Schema.Picker)
-      .prop("hasPMOAdminEditCapability", S.boolean())
```

Comment:
i think we need this, maybe we should discuss why it was added, so you don't remove this, just because